variables.tf

```diff
@@ -1095,6 +1095,18 @@ variable "ecs_requires_compatibilities" {
   default     = ["EC2", "FARGATE"]
 }
 
+variable "s3_versioning_expiration_days" {
+  description = "Number of days (TTL) before objects are expired. Bucket need to have versioning enabled."
+  type        = number
+  default     = 7
+}
+
+variable "s3_versioning_max_versions_retained" {
+  description = "Number of noncurrent versions Amazon S3 will retain. Must be a non-zero positive integer. Bucket need to have versioning enabled."
+  type        = number
+  default     = 3
```

Comment:
This should be defaulted off... we don't want to force keeping a number of versions, we want all the versions to be deleted after 7 days by default. If both are set, then it will keep the max versions regardless of the days, which means we will end up keeping some of the deleted data forever (at least this is what I believe I tested and saw that it always retained up to the max versions).